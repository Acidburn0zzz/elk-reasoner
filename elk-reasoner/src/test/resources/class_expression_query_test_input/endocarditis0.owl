Prefix(:=<owlapi:ontology#ont5#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<owlapi:ontology#ont5>

Declaration(Class(<http://www.test.com/schema#BodyValve>))
Declaration(Class(<http://www.test.com/schema#Disease>))
Declaration(Class(<http://www.test.com/schema#Endocarditis>))
Declaration(Class(<http://www.test.com/schema#Endocardium>))
Declaration(Class(<http://www.test.com/schema#Heart>))
Declaration(Class(<http://www.test.com/schema#HeartValve>))
Declaration(Class(<http://www.test.com/schema#Inflammation>))
Declaration(Class(<http://www.test.com/schema#IsStateNeedsTreatment>))
Declaration(Class(<http://www.test.com/schema#Pericarditis>))
Declaration(Class(<http://www.test.com/schema#Pericardium>))
Declaration(Class(<http://www.test.com/schema#Tissue>))
Declaration(ObjectProperty(<http://www.test.com/schema#acts-on>))
Declaration(ObjectProperty(<http://www.test.com/schema#contained-in>))
Declaration(ObjectProperty(<http://www.test.com/schema#has-location>))
Declaration(ObjectProperty(<http://www.test.com/schema#part-of>))
############################
#   Object Properties
############################

# Object Property: <http://www.test.com/schema#part-of> (<http://www.test.com/schema#part-of>)

SubObjectPropertyOf(<http://www.test.com/schema#part-of> <http://www.test.com/schema#contained-in>)


############################
#   Classes
############################

# Class: <http://www.test.com/schema#Endocarditis> (<http://www.test.com/schema#Endocarditis>)

SubClassOf(<http://www.test.com/schema#Endocarditis> ObjectIntersectionOf(<http://www.test.com/schema#Inflammation> ObjectSomeValuesFrom(<http://www.test.com/schema#has-location> <http://www.test.com/schema#Endocardium>)))

# Class: <http://www.test.com/schema#Endocardium> (<http://www.test.com/schema#Endocardium>)

SubClassOf(<http://www.test.com/schema#Endocardium> ObjectIntersectionOf(<http://www.test.com/schema#Tissue> ObjectSomeValuesFrom(<http://www.test.com/schema#contained-in> <http://www.test.com/schema#HeartValve>)))

# Class: <http://www.test.com/schema#HeartValve> (<http://www.test.com/schema#HeartValve>)

SubClassOf(<http://www.test.com/schema#HeartValve> ObjectIntersectionOf(<http://www.test.com/schema#BodyValve> ObjectSomeValuesFrom(<http://www.test.com/schema#part-of> <http://www.test.com/schema#Heart>)))

# Class: <http://www.test.com/schema#Inflammation> (<http://www.test.com/schema#Inflammation>)

SubClassOf(<http://www.test.com/schema#Inflammation> ObjectIntersectionOf(<http://www.test.com/schema#Disease> ObjectSomeValuesFrom(<http://www.test.com/schema#acts-on> <http://www.test.com/schema#Tissue>)))

# Class: <http://www.test.com/schema#Pericarditis> (<http://www.test.com/schema#Pericarditis>)

SubClassOf(<http://www.test.com/schema#Pericarditis> ObjectIntersectionOf(<http://www.test.com/schema#Inflammation> ObjectSomeValuesFrom(<http://www.test.com/schema#has-location> <http://www.test.com/schema#Pericardium>)))

# Class: <http://www.test.com/schema#Pericardium> (<http://www.test.com/schema#Pericardium>)

SubClassOf(<http://www.test.com/schema#Pericardium> ObjectIntersectionOf(<http://www.test.com/schema#Tissue> ObjectSomeValuesFrom(<http://www.test.com/schema#contained-in> <http://www.test.com/schema#Heart>)))


SubClassOf(ObjectIntersectionOf(ObjectIntersectionOf(<http://www.test.com/schema#Disease> ObjectSomeValuesFrom(<http://www.test.com/schema#has-location> <http://www.test.com/schema#Heart>)) ObjectSomeValuesFrom(<http://www.test.com/schema#has-location> <http://www.test.com/schema#HeartValve>)) <http://www.test.com/schema#IsStateNeedsTreatment>)
SubObjectPropertyOf(ObjectPropertyChain(<http://www.test.com/schema#has-location> <http://www.test.com/schema#contained-in>) <http://www.test.com/schema#has-location>)
)