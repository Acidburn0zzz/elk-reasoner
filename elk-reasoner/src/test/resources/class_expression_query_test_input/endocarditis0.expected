Prefix(:=<owlapi:ontology#ont6#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<owlapi:ontology#ont6>

Declaration(Class(<http://www.test.com/schema#Disease>))
Declaration(Class(<http://www.test.com/schema#Endocarditis>))
Declaration(Class(<http://www.test.com/schema#Heart>))
Declaration(Class(<http://www.test.com/schema#Pericarditis>))
Declaration(ObjectProperty(<http://www.test.com/schema#has-location>))

############################
#   Classes
############################

# Class: <http://www.test.com/schema#Endocarditis> (<http://www.test.com/schema#Endocarditis>)

SubClassOf(<http://www.test.com/schema#Endocarditis> ObjectIntersectionOf(<http://www.test.com/schema#Disease> ObjectSomeValuesFrom(<http://www.test.com/schema#has-location> <http://www.test.com/schema#Heart>)))

# Class: <http://www.test.com/schema#Pericarditis> (<http://www.test.com/schema#Pericarditis>)

SubClassOf(<http://www.test.com/schema#Pericarditis> ObjectIntersectionOf(<http://www.test.com/schema#Disease> ObjectSomeValuesFrom(<http://www.test.com/schema#has-location> <http://www.test.com/schema#Heart>)))



SubClassOf(ObjectIntersectionOf(<http://www.test.com/schema#Disease> ObjectSomeValuesFrom(<http://www.test.com/schema#has-location> <http://www.test.com/schema#Heart>)) <http://www.test.com/schema#Disease>)
)